starlite/config.py

```diff
@@ -141,6 +142,20 @@ def dict(self, *args, **kwargs) -> Dict[str, Any]:  # type: ignore[no-untyped-de
         return super().dict(*args, **kwargs)
 
 
+class CSRFConfig(BaseModel):
+    """Class containing CSRF configuration."""
+
+    secret: str
+    cookie_name: str = "csrftoken"
+    cookie_path: str = "/"
+    header_name: str = "x-csrftoken"
+    cookie_secure: bool = False
+    cookie_httponly: bool = False
+    cookie_samesite: str = "Lax"
```

Comment:
i think its actually `lax` rather. Also, typing should be I think `Literal["lax", "strict", "none"]`:

```suggestion
    cookie_samesite: Literal["lax", "strict", "none"] = "lax"
```

Also make sure to import `Literal` from `typing_extensions` for backwards compat with py 3.7.

Reply:
Done!